sbi/inference/posteriors/sbi_posterior.py

```diff
@@ -198,9 +212,14 @@ def sample(
         r"""
         Return samples from posterior distribution $p(\theta|x)$.
 
+        Samples are obtained either with rejection sampling or MCMC. SNPE can use
+        rejection sampling and MCMC (which can help to deal with strong leakage). SNL
```

Comment:
maybe add the distinction between posterior and L again: 
SNPE can use rejection sampling and MCMC (which ...) on the posterior. SNL and SRE are restricted to sampling with MCMC via the likelihood (ratio). 

Not sure though.

Reply:
I think it's fine the way it is here. The user need not care so much about whether the likelihood (ratio) is used to generate the samples.